### PR TITLE
[Keymap] add noroadsleft userspace; add and update keymaps

### PR DESCRIPTION
**Pull request by noroadsleft**
## Description

[title]

## Types of Changes

- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: users/noroadsleft/noroadsleft.c [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            } else {
+                if (get_mods() & MOD_MASK_RALT) {
+                    unregister_code(keycode + 0x2E);
+                } else {
+                    unregister_code(keycode);
+                }
+            }
+            return false;
+    }  // switch()
+    return process_record_keymap(keycode, record);
```
Comment (drashna) [inline users/noroadsleft/noroadsleft.c:149]: Personally, I like having this checked at top, before the switch.  Eg:

```c 
if (!process_record_keymap(keycode, record)) return false;
```
It's a bit more inline with the core.

**Review by drashna — APPROVED** [review]

File: keyboards/kbdfans/kbd75/keymaps/noroadsleft/config.h [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#undef MANUFACTURER
+#undef PRODUCT
+#undef DESCRIPTION
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd75/keymaps/noroadsleft/config.h:21]:
```suggestion
```

File: keyboards/kbdfans/kbd75/keymaps/noroadsleft/keymap.c [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "noroadsleft.h"
+//#include "version.h"
+#include <sendstring_dvorak.h>
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd75/keymaps/noroadsleft/keymap.c:19]:
```suggestion
#include "sendstring_dvorak.h"
```

File: keyboards/kc60/keymaps/noroadsleft/keymap.c [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "noroadsleft.h"
 #include <sendstring_dvorak.h>
 //#include <sendstring_colemak.h>
 #include <print.h>
```
Comment (fauxpark) [inline keyboards/kc60/keymaps/noroadsleft/keymap.c:20]:
```suggestion
#include "sendstring_dvorak.h"
//#include "sendstring_colemak.h"
#include "print.h"
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by Erovia — APPROVED** [review]

**Pull request merged**